app/styles/_our-vision-section.scss

```diff
@@ -0,0 +1,42 @@
+section.our-vision {
+  padding: 50px;
+  h2 {
+    font-size: 29px;
+    padding-bottom: 40px;
+    line-height: 30px;
+    text-align: center;
+  }
+  .col-md-6 {
+    background-color: white;
+    text-align: center;
+    i {
+      font-size: 40px;
+      background-color: $green;
+      padding: 30px;
+      margin-bottom: 10px;
+      &.fas {
+        background-color: $green;
+        border-radius: 50%;
+        align-items: center;
+        justify-content: space-around;
+        color: $white;
+      }
+    }
+    p {
+      line-height: 28px;
+    }
+    h4 {
+      justify-content: space-around;
+      align-items: center;
+      padding-top: 20px;
+      color: $black;
+      font-size: 2em;
+      font-weight: 400;
+      padding-bottom: 20px;
+    }
+    .vision-item {
+      width: 90%;
```

Comment:
I personally am not a big fan of using percentages. I understand that they are more responsive, but I prefer to use mixins  as they're a bit more predictable